src/api/windows.py

```diff
@@ -142,7 +141,7 @@ def process_result_dynamic(result):
     end = end_point - (end_point % p)
 
     # Generate sample space from period
-    x = np.linspace(0, end, int(end / p))
+    x = np.linspace(0, end, int(end / p) + 1)
```

Comment:
Maybe (?) this should have been put into it's own patch commit, but it probably doesn't matter.